src/units.h

```diff
@@ -308,11 +309,11 @@ inline constexpr double to_liter( const volume &v )
 // Don't use in new code! Use one of the from_* functions instead.
 static constexpr volume legacy_volume_factor = from_milliliter( 250 );
 
-class mass_in_gram_tag
+class mass_in_milligram_tag
 {
 };
 
-using mass = quantity<int, mass_in_gram_tag>;
+using mass = quantity<std::int_fast64_t, mass_in_milligram_tag>;
```

Comment:
```suggestion
using mass = quantity<std::int64_t, mass_in_milligram_tag>;
```
We're already using `int64_t` in various places, so IMO there's little benefit to using `int_fast64_t` here.  It's just more typing for the same result.